lib/child_process.js

```diff
@@ -151,6 +150,8 @@ exports.execFile = function(file /*, args, options, callback*/) {
     pos++;
   }
 
+  options.killSignal = validateKillSignal(options.killSignal);
```

Comment:
Could you move this down to around [here](https://github.com/thefourtheye/io.js/blob/394216c95d189f52a3d2583bbf6272b933941310/lib/child_process.js#L163-L167).